### =========================================================================
### preciseTADhub metadata
### -------------------------------------------------------------------------
###

meta <- data.frame(
    Title = c("CHR1_GM12878_5kb_Arrowhead.rds",
              "CHR1_GM12878_10kb_Peakachu.rds",
              "CHR1_K562_5kb_Arrowhead.rds",
              "CHR1_K562_10kb_Peakachu.rds",
              "CHR2_GM12878_5kb_Arrowhead.rds",
              "CHR2_GM12878_10kb_Peakachu.rds",
              "CHR2_K562_5kb_Arrowhead.rds",
              "CHR2_K562_10kb_Peakachu.rds",
              "CHR3_GM12878_5kb_Arrowhead.rds",
              "CHR3_GM12878_10kb_Peakachu.rds",
              "CHR3_K562_5kb_Arrowhead.rds",
              "CHR3_K562_10kb_Peakachu.rds",
              "CHR4_GM12878_5kb_Arrowhead.rds",
              "CHR4_GM12878_10kb_Peakachu.rds",
              "CHR4_K562_5kb_Arrowhead.rds",
              "CHR4_K562_10kb_Peakachu.rds",
              "CHR5_GM12878_5kb_Arrowhead.rds",
              "CHR5_GM12878_10kb_Peakachu.rds",
              "CHR5_K562_5kb_Arrowhead.rds",
              "CHR5_K562_10kb_Peakachu.rds",
              "CHR6_GM12878_5kb_Arrowhead.rds",
              "CHR6_GM12878_10kb_Peakachu.rds",
              "CHR6_K562_5kb_Arrowhead.rds",
              "CHR6_K562_10kb_Peakachu.rds",
              "CHR7_GM12878_5kb_Arrowhead.rds",
              "CHR7_GM12878_10kb_Peakachu.rds",
              "CHR7_K562_5kb_Arrowhead.rds",
              "CHR7_K562_10kb_Peakachu.rds",
              "CHR8_GM12878_5kb_Arrowhead.rds",
              "CHR8_GM12878_10kb_Peakachu.rds",
              "CHR8_K562_5kb_Arrowhead.rds",
              "CHR8_K562_10kb_Peakachu.rds",
              "CHR10_GM12878_5kb_Arrowhead.rds",
              "CHR10_GM12878_10kb_Peakachu.rds",
              "CHR10_K562_5kb_Arrowhead.rds",
              "CHR10_K562_10kb_Peakachu.rds",
              "CHR11_GM12878_5kb_Arrowhead.rds",
              "CHR11_GM12878_10kb_Peakachu.rds",
              "CHR11_K562_5kb_Arrowhead.rds",
              "CHR11_K562_10kb_Peakachu.rds",
              "CHR12_GM12878_5kb_Arrowhead.rds",
              "CHR12_GM12878_10kb_Peakachu.rds",
              "CHR12_K562_5kb_Arrowhead.rds",
              "CHR12_K562_10kb_Peakachu.rds",
              "CHR13_GM12878_5kb_Arrowhead.rds",
              "CHR13_GM12878_10kb_Peakachu.rds",
              "CHR13_K562_5kb_Arrowhead.rds",
              "CHR13_K562_10kb_Peakachu.rds",
              "CHR14_GM12878_5kb_Arrowhead.rds",
              "CHR14_GM12878_10kb_Peakachu.rds",
              "CHR14_K562_5kb_Arrowhead.rds",
              "CHR14_K562_10kb_Peakachu.rds",
              "CHR15_GM12878_5kb_Arrowhead.rds",
              "CHR15_GM12878_10kb_Peakachu.rds",
              "CHR15_K562_5kb_Arrowhead.rds",
              "CHR15_K562_10kb_Peakachu.rds",
              "CHR16_GM12878_5kb_Arrowhead.rds",
              "CHR16_GM12878_10kb_Peakachu.rds",
              "CHR16_K562_5kb_Arrowhead.rds",
              "CHR16_K562_10kb_Peakachu.rds",
              "CHR17_GM12878_5kb_Arrowhead.rds",
              "CHR17_GM12878_10kb_Peakachu.rds",
              "CHR17_K562_5kb_Arrowhead.rds",
              "CHR17_K562_10kb_Peakachu.rds",
              "CHR18_GM12878_5kb_Arrowhead.rds",
              "CHR18_GM12878_10kb_Peakachu.rds",
              "CHR18_K562_5kb_Arrowhead.rds",
              "CHR18_K562_10kb_Peakachu.rds",
              "CHR19_GM12878_5kb_Arrowhead.rds",
              "CHR19_GM12878_10kb_Peakachu.rds",
              "CHR19_K562_5kb_Arrowhead.rds",
              "CHR19_K562_10kb_Peakachu.rds",
              "CHR20_GM12878_5kb_Arrowhead.rds",
              "CHR20_GM12878_10kb_Peakachu.rds",
              "CHR20_K562_5kb_Arrowhead.rds",
              "CHR20_K562_10kb_Peakachu.rds",
              "CHR21_GM12878_5kb_Arrowhead.rds",
              "CHR21_GM12878_10kb_Peakachu.rds",
              "CHR21_K562_5kb_Arrowhead.rds",
              "CHR21_K562_10kb_Peakachu.rds",
              "CHR22_GM12878_5kb_Arrowhead.rds",
              "CHR22_GM12878_10kb_Peakachu.rds",
              "CHR22_K562_5kb_Arrowhead.rds",
              "CHR22_K562_10kb_Peakachu.rds"),
    Description = c("Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR1 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR1 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR1 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR1 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR2 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR2 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR2 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR2 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR3 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR3 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR3 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR3 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR4 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR4 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR4 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR4 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR5 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR5 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR5 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR5 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR6 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR6 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR6 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR6 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR7 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR7 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR7 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR7 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR8 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR8 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR8 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR8 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR10 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR10 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR10 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR10 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR11 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR11 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR11 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR11 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR12 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR12 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR12 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR12 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR13 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR13 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR13 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR13 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR14 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR14 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR14 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR14 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR15 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR15 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR15 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR15 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR16 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR16 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR16 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR16 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR17 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR17 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR17 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR17 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR18 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR18 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR18 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR18 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR19 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR19 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR19 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR19 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR20 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR20 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR20 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR20 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR21 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR21 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR21 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR21 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR22 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR22 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for GM12878",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR22 as a holdout chromosome, Arrowhead ground truth TAD boundaries, at 5 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562",
                    "Pre-trained model built using TADrandomForest from the preciseTAD R package with CHR22 as a holdout chromosome, Peakachu ground truth TAD boundaries, at 10 kb resolution, with random under-sampling, and distance-type features from CTCF, RAD21, SMC3, and ZNF143 ChIP-seq peaks for K562"),
    BiocVersion = c(rep("3.12", 84)),
    Genome = c(rep("hg19", 84)),
    SourceType = c(rep("RDS", 84)),
    SourceUrl = "https://onedrive.live.com/?authkey=%21AEdP2%5FgyT%5FY0zSs&id=5BE46CBE2B039B19%2146565&cid=5BE46CBE2B039B19",
    SourceVersion = "Oct 16 2020",
    Species = "Homo sapiens",
    TaxonomyId = 9606,
    Coordinate_1_based = TRUE,
    DataProvider = "preciseTAD",
    Maintainer = "Mikhail Dozmorov <mikhail.dozmorov@vcuhealth.org>",
    RDataClass = c(rep("list", 84)),
    DispatchClass = c(rep("Rds",84)),
    RDataPath = c("preciseTADhub/CHR1_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR1_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR1_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR1_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR2_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR2_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR2_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR2_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR3_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR3_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR3_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR3_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR4_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR4_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR4_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR4_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR5_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR5_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR5_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR5_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR6_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR6_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR6_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR6_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR7_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR7_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR7_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR7_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR8_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR8_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR8_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR8_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR10_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR10_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR10_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR10_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR11_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR11_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR11_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR11_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR12_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR12_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR12_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR12_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR13_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR13_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR13_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR13_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR14_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR14_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR14_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR14_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR15_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR15_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR15_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR15_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR16_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR16_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR16_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR16_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR17_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR17_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR17_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR17_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR18_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR18_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR18_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR18_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR19_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR19_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR19_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR19_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR20_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR20_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR20_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR20_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR21_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR21_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR21_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR21_K562_10kb_Peakachu.rds",
                  "preciseTADhub/CHR22_GM12878_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR22_GM12878_10kb_Peakachu.rds",
                  "preciseTADhub/CHR22_K562_5kb_Arrowhead.rds",
                  "preciseTADhub/CHR22_K562_10kb_Peakachu.rds"),
    Tags = "",
    Notes = "")

write.csv(meta, file="inst/extdata/metadata.csv", row.names=FALSE)
